src/styles/card.scss

```diff
@@ -0,0 +1,97 @@
+body {
+  margin: 0;
+  display: flex;
+  justify-content: center;
+}
+
+.card {
+  display: block;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    padding: 32px 19px 36px;
+  }
+
+  &__product {
+    display: flex;
+    flex-direction: column;
+    padding: 0 16px;
+    margin-bottom: 16px;
+
+    &-name {
+      margin: 0;
+      font-family: $font-Roboto;
+      color: $main-font-color;
+      font-size: 12px;
+      font-weight: 500;
+      line-height: 18px;
+    }
+
+    &-code {
+      margin: 0;
+      margin-top: 4px;
+      color: $secondary-font-color;
+      font-size: 10px;
+      font-family: $font-Roboto;
+      font-weight: 400;
+      line-height: 14px;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+
+    &-text {
+      margin: 0;
+      font-family: $font-Roboto;
+      color: $secondary-font-color;
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+      padding-left: 16px;
+    }
+
+    &-number {
+      margin: 0;
+      padding-right: 16px;
```

Comment:
it's better to add left and right padding 16px for the whole card instead of doing it in every element